src/needle.ts

```diff
@@ -109,6 +115,13 @@ export async function makeRequest<T = void>(
 
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
     const errorMessage = response?.body?.error;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access
+    const errors = response?.body?.errors;
+
+    if (isJsonApiErrors(errors)) {
+      jsonApiError = errors[0];
```

Comment:
`nit`: This behaviour (only handling first error) is not documented anywhere.
I don't see cases where we have more than one error from the API TBH, but know the JSON API requires arrays.
If someone were to change the API to actually return more than one error they will probably not realise the client can only handle one though.

Reply:
Yeah I guess this is a limitation right now, where would you think we can document it?

Reply:
The good thing is if we have more than 1 error, nothing breaks. They will be ignored.

Perhaps a single comment from the service that throws the error?